ocfweb/docs/templates/docs/partials/host.html

```diff
@@ -3,6 +3,7 @@
 
 <div class="row host host-{{host.type}}">
     <div class="col-sm-3">
+        <a id="{{host.hostname}}" class="server-anchor"></a>
```

Comment:
Could this link be on the hostname below so that when the name is clicked on it then changes the URL to go directly to that host? Seems like it would be a nice way to get a link to the host.